### PR TITLE
Folderise simorgh app lib

### LINKED ISSUE #1963: Folderise simorgh app lib
**Is your feature request related to a problem? Please describe.**
The folder `src/app/lib` in simorgh has lot of files https://github.com/bbc/simorgh/tree/latest/src/app/lib

**Describe the solution you'd like**
Move from the structure
```
lib
- styledGrid.jsx
- styledGrid.test.jsx
- styledGrid.stories.jsx
```
to
```
lib
- styledGrid
-- index.jsx
-- index.test.jsx
-- index.stories.jsx
```
With all files in the folder. If you want to do this with other parts of simorgh, be my guest.

**Testing notes**
Requires a regression.

Dev insight: Regression

### PR DESCRIPTION
**Pull request by thekp**
Resolves #1963

**Overall change:** _Organise files into folders that define it's purpose_

**Code changes:**

- _update imports_
- _update snapshots_

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Tests added for new features
- [ ] Test engineer approval
- [ ] I have followed [the merging checklist](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/MERGE_PROCESS.md) and this is ready to merge.

### CODE REVIEW

**Comment by thekp** [comment]
FYI: the code climate issues, are already present on the current latest.
e.g: https://codeclimate.com/github/bbc/simorgh/pull/1680

**Review by j-pendlebury — APPROVED** [review]
image
Looks good 👍 
Let's ignore codeclimate, it's not being very reliable today.

File: src/app/lib/styledGrid/index.test.jsx [hunk 1]
```diff
@@ -9,7 +9,7 @@ import {
   NestedGridItemSmall,
   NestedGridItemMedium,
   NestedGridItemLarge,
-} from './styledGrid';
+} from './index';
```
Comment (sareh) [inline src/app/lib/styledGrid/index.test.jsx]:
```suggestion
} from '.';
```

File: src/app/lib/styledGrid/index.stories.jsx [hunk 2]
```diff
@@ -10,7 +10,7 @@ import {
   NestedGridItemSmall,
   NestedGridItemMedium,
   NestedGridItemLarge,
-} from './styledGrid';
+} from './index';
```
Comment (sareh) [inline src/app/lib/styledGrid/index.stories.jsx]:
```suggestion
} from '.';
```

File: src/app/lib/globalStyles/index.test.jsx [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { render } from '@testing-library/react';
-import { ServiceContext } from '../contexts/ServiceContext';
-import GlobalStyles from './globalStyles';
+import { ServiceContext } from '../../contexts/ServiceContext';
+import GlobalStyles from './index';
```
Comment (sareh) [inline src/app/lib/globalStyles/index.test.jsx]:
```suggestion
import GlobalStyles from '.';
```

**Review by sareh — COMMENTED** [review]
Looks good. Just tiny suggestions to tidy imports. :)

**Comment by thekp** [comment]
> Looks good. Just tiny suggestions to tidy imports. :)

thanks @sareh . I've made the changes now 😸

**Review by sareh — APPROVED** [review]
👍 Looks great, thanks @thekp!

**Comment by staylos92** [comment]
LGTM 👍

**Pull request merged**